### ISSUE TITLE
Insufficient contrast ratio for gray text

### ISSUE THREAD

**Issue by meganvilla** [OpenSpace/OpenSpace#3127]
Different shades of gray color are used for placeholder text, labels or entered text in an  input. On several instances, the gray color has insufficient contrast ratio against the  black page background or other gray shades, making the text hard to read for users  with low vision. 

Some examples are included in the following data table:

Element | Font Color | Background | Contrast Ratio
-- | -- | -- | --
"Search" placeholder | #757575 | #161616 | 3.9:1
Slider labels when placed on the bar for the selected value | #999 | #3A3A3A | 4.1:1
Text typed in comboboxes | #333 | #222 | 1.3:1

**Suggested Solution** 
Ensure a contrast ratio of at least 4.5:1 is achieved for all normal-sized text.

**Comment by ylvaselling** [comment]
It would be great to have a general design guideline for these things, with set colors for different parts of the UI, fonts, margins etc. Imo it would be fantastic to have a designer create a guideline we can use everywhere (documentation, UI, webpage, youtube channel etc etc) for visual consistency. It would also make it easier for developers to just follow these guidelines rather than create our own designs.

**Comment by alexanderbock** [comment]
The colors have been redesigned to have higher contrast in e3f5b83f7476c8507dccd45ab98e4731504af532
